content/feature-guides/rate-limiting.md

```diff
@@ -19,4 +19,4 @@ You can refer to the rate limit for each endpoint in the table below:
 
 ## STX faucet
 
-The Stacks faucet rate limits depend on the type of request. For stacking requests, there is a limit of **1 request per 2 days**. In case of regular Stacks faucet requests, the limits are set to **1 request per minute**.
+The Stacks faucet rate limits depend on the type of request. For stacking requests, there is a limit of **1 request per 2 days**. In case of regular Stacks faucet requests, the limits are set to **1 request per day**.
```

Comment:
I don't think we actually have a different rate limit depending on whether it's a stacking request or not. It's always 1 request per day.